packages/middleware-expect-continue/src/index.ts

```diff
@@ -9,14 +9,24 @@ import {
 } from "@aws-sdk/types";
 import { HttpRequest } from "@aws-sdk/protocol-http";
 
-export function addExpectContinueMiddleware(): BuildMiddleware<any, any> {
+interface PreviouslyResolved {
+  runtime: string;
```

Comment:
use `RuntimeTarget` instead of string
```suggestion
  runtime: RuntimeTarget;
```

where:
```typescript
import {
  RuntimeTarget
} from "@aws-sdk/build-types";
```

We can change Runtime target to [Enum](https://www.typescriptlang.org/docs/handbook/enums.html) if required, in a separate PR

Reply:
@aws-sdk/build-types is probably going away, so I'll leave it as-is for now.